gazelle/pythonconfig/pythonconfig.go

```diff
@@ -50,6 +50,10 @@ const (
 	// GenerationMode represents the directive that controls the target generation
 	// mode. See below for the GenerationModeType constants.
 	GenerationMode = "python_generation_mode"
+	// GenerationModePerFileIncludeInit represents the directive that augments
+	// the "per_file" GenerationMode by including the package's __init__.py file.
+	// This is a boolean directive.
+	GenerationModePerFileIncludeInit = "python_generation_mode_per_file_include_init"
```

Comment:
Could you also document this in `gazelle/README.md` please?

Reply:
Done. Thanks.